src/components/tx-flow/flows/SafeAppsTx/index.tsx

```diff
@@ -12,14 +12,20 @@ export type SafeAppsTxParams = {
   params?: SendTransactionRequestParams
 }
 
-const SafeAppsTxFlow = ({ data }: { data: SafeAppsTxParams }) => {
+const SafeAppsTxFlow = ({
```

Comment:
Not sure why but I didn't see a `Transaction created` when I created one via CSV Airdrop safe app. Only `Propose Transaction` and `On-chain interaction`